src/main/webapp/step3.html

```diff
@@ -3,54 +3,16 @@
         <meta charset="UTF-8">
         <title>Home</title>
         <link rel="stylesheet" href="style.css">
+        <script type="module" src="step3-startup.js"></script>
     </head>
     <body>
         <nav>
             <!-- TODO: Add the navigation - common to all steps (@tzavidas) -->
         </nav>
         <div class="content">
             <h2 class="title">Iterable Overview</h2>
-            <div class="main-inteface">
-                <!-- TODO: Replace this with dynamically generated code in JavaScript (@tzavidas) -->
```

Comment:
Same TODO format rule as above

Reply:
Thanks for the input! This does not apply anymore though, because I have removed the code!